dGame/Character.h

```diff
@@ -364,6 +364,30 @@ class Character {
      */
 	void SetAnnouncementMessage(const std::string& value) { m_AnnouncementMessage = value; }
 
+    /**
+     * Gets the subject of a mail message that a character made (reserved for GMs)
+     * @return the title of the announcement a character made
+     */
+	const std::string& GetMailSubject() const { return m_MailSubject; }
+
+    /**
+     * Sets the subject of a mail message a character will make (reserved for GMs)
+     * @param value the subject to set
+     */
+	void SetMailSubject(const std::string& value) { m_MailSubject = value; }
+
+    /**
+     * Gets the body of a mail message a character made (reserved for GMs)
+     * @return the body of the mail
+     */
+	const std::string& GetMailBody() const { return m_MailBody; }
+
+    /**
+     * Sets the body of an mail message to make (reserved for GMs)
+     * @param value the body of the mail message
+     */
+	void SetMailBody(const std::string& value) { m_MailBody = value; }
+
```

Comment:
I'm assuming these changes were accidentally pushed to this branch? These belong in a separate PR, not one with leaderboards bundled.

Reply:
You are correct. I just reverted these, sorry for the false alarm. This was another project I've been working on this morning.

Reply:
Ah, saw your comment about mistakenly pushing that change. Resolving this one.